package.json

```diff
@@ -36,34 +35,33 @@
-    "visual-testing-app:build": "yarn --cwd visual-testing-app build",
-    "visual-testing-app:serve": "yarn --cwd visual-testing-app serve",
-    "template-starter:build": "yarn --cwd application-templates/starter build",
-    "template-starter:start": "yarn --cwd application-templates/starter start",
-    "template-starter:start:prod:local": "yarn --cwd application-templates/starter start:prod:local",
-    "prebuild": "lerna run prebuild --no-private",
-    "build": "preconstruct build && NODE_ENV=production lerna run build --no-private",
-    "build:watch": "preconstruct watch & NODE_ENV=development lerna run --no-private --parallel build:bundles:watch",
-    "build:website": "yarn build && yarn --cwd website build && yarn --cwd website-components-playground build",
+    "playground:build": "yarn manypkg run playground build",
+    "playground:start": "yarn manypkg run playground start",
+    "playground:start:prod:local": "yarn manypkg run playground start:prod:local",
+    "visual-testing-app:start": "yarn manypkg run visual-testing-app start",
+    "visual-testing-app:build": "yarn manypkg run visual-testing-app build",
+    "visual-testing-app:serve": "yarn manypkg run visual-testing-app serve",
+    "template-starter:build": "yarn manypkg run application-template-starter build",
+    "template-starter:start": "yarn manypkg run application-template-starter start",
+    "template-starter:start:prod:local": "yarn manypkg run application-template-starter start:prod:local",
+    "build": "preconstruct build && NODE_ENV=production yarn workspaces run build",
+    "build:watch": "preconstruct watch & NODE_ENV=development yarn manypkg run mc-html-template build:bundles:watch & & NODE_ENV=development yarn manypkg run mc-scripts build:bundles:watch",
+    "build:website": "yarn build && yarn manypkg run @commercetools-website/custom-applications build && yarn manypkg @commercetools-website/components-playground build",
     "now-build": "yarn build:website && rm -rf public && mv website/public public",
     "labels:sync": "github-labels sync",
     "generate-types:mc": "./scripts/graphql-cli.sh codegen --project mc",
     "generate-types:ctp": "./scripts/graphql-cli.sh codegen --project ctp",
     "generate-types:settings": "./scripts/graphql-cli.sh codegen --project settings",
-    "generate-types:proxy": "./scripts/graphql-cli.sh codegen --project proxy",
+    "generate-types:proxy": "./scripts/graphql-cli.sh codegen --project iproxy",
     "generate-types": "yarn generate-types:mc && yarn generate-types:ctp && yarn generate-types:settings && yarn generate-types:proxy",
     "versions:uikit": "manypkg upgrade @commercetools-uikit",
     "versions:docskit": "manypkg upgrade @commercetools-docs"
   },
   "workspaces": {
     "packages": [
-      "application-templates/*",
-      "packages-backend/*",
       "packages/*",
+      "packages-backend/*",
+      "application-templates/*",
```

Comment:
This ordering actually determines in what order yarn runs things. Which makes total sense. We want to build our actual packages first before using them in e.g. the template.

Reply:
Ah didn't know about that. Thanks!